BackofficeBundle/Resources/public/coffee/orchestraLib.coffee

```diff
@@ -159,7 +159,115 @@ callback_tinymce_init = null
 
 activateTinyMce = (view, textarea) ->
   do (view, textarea) ->
+    tinymce.html2bbcodeOrchestra =
+      '<img.*?src="(.*?)".*?alt="(.*?)".*?title="(.*?)"\\/>' : '[img alt="$2" title="$3"]$1[/img]',
```

Comment:
Nice idea, each bundle should be able to add it's own tag to the main configuration